third_party/s2n-bignum/include/s2n-bignum_aws-lc.h

```diff
@@ -246,3 +246,81 @@ extern void bignum_copy_row_from_table_16_neon (uint64_t *z, const uint64_t *tab
 // Input table[height*32]; output z[32]
 extern void bignum_copy_row_from_table_32_neon (uint64_t *z, const uint64_t *table,
         uint64_t height, uint64_t idx);
+
+#if !defined(BORINGSSL_FIPS)
+// Reduction is modulo the order of the curve25519/edwards25519 basepoint,
+// which is n_25519 = 2^252 + 27742317777372353535851937790883648493.
+// Reduce modulo basepoint order, z := x mod n_25519
+// Input x[k]; output z[4]
+extern void bignum_mod_n25519(uint64_t z[static 4], uint64_t k, uint64_t *x);
```

Comment:
This is a data point for not making this a submodule, and leaving it as a subtree. Its much more complicated if we are patching the sources rather then using them as-is.

Reply:
Yes - something that would need to be fixed before actually switching over...